core/source/api/opencmiss/zinc/fieldimage.h

```diff
@@ -481,14 +481,74 @@ ZINC_API int cmzn_field_image_set_wrap_mode(cmzn_field_image_id image_field,
  * Gets the property named in the given property string from the given field image.
  * The returned string must be deallocated by the receiver.
  *
- * @param	image   	The image field.
+ * @param	image_field   	The image field.
  * @param	property	The property to retrieve.
  *
  * @return	null if it fails, else an allocated string containing the properties value.
  */
-ZINC_API char *cmzn_field_image_get_property(cmzn_field_image_id image,
+ZINC_API char *cmzn_field_image_get_property(cmzn_field_image_id image_field,
 	const char* property);
 
+/**
+ * Get the number of bytes per component of this image field.
+ *
+ * @param image_field 	The image field.
+ *
+ * @return  number of bytes of this image field.
+ */
+ZINC_API int cmzn_field_image_get_number_of_bytes_per_component(
+	cmzn_field_image_id image_field);
+
+/**
+ * Set the number of component of the image. The number of bytes will only be set
+ *  if no image has been set into this image field.
+ *
+ * @param image_field 	The image field.
+ * @param number_of_bytes  Number of bytes to be set for this field image.
+ *
+ * @return  Status CMZN_OK on success, any other value on failure.
+ */
+ZINC_API int cmzn_field_image_set_number_of_bytes_per_component(
+	cmzn_field_image_id image_field, int number_of_bytes);
+
+/**
+ * Get the pixel format per componentof this image field.
+ *
+ * @param image_field 	The image field.
+ *
+ * @return  pixel format of this image field.
+ */
+ZINC_API enum cmzn_streaminformation_image_pixel_format
```

Comment:
Can you replace this (and set_pixel_format) to use a new enumeration:
enum cmzn_field_image_pixel_format
enum cmzn_streaminformation_image_pixel_format can be set to the same values as this enumeration.
This is then consistent with having enum PixelFormat in class FieldImage below.

Reply:
Sure.